apps/artemis-web/components/hijack-table/hijack-table.tsx

```diff
@@ -605,7 +605,7 @@ const HijackTableComponent = (props) => {
           setStateValues({
             ...stateValues,
             [key]:
-              isNumeric(filters[key].filterVal) || key === 'rpki_status'
+              isNumeric(filters[key].filterVal) || key === 'rpki_status' || key === 'type'
```

Comment:
does this fix work for best match types, like 'S', 'S|0' etc., or should we put the entire type? This worked initially with best (partial) match (LIKE).

Reply:
no need to put the entire type! It is a partial match!